elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -45,7 +51,7 @@ $variables: (
   position: relative;
   z-index: 0;
   vertical-align: middle;
-  transition: padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
+  //transition: padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
```

Comment:
@castastrophe  when i commented the other one out, it stopped animating. So i commented this one out (to test) and it seems to still work.. with no flicker. I'll send it out for another round of "can anyone see a flicker"